TCAT/Controllers/SearchResultsTableViewController.swift

```diff
@@ -187,25 +199,25 @@ class SearchResultsTableViewController: UITableViewController, UISearchBarDelega
         switch sections[indexPath.section].type {
         case .cornellDestination:
             itemType = .cornellDestination
-        case .recentSearches, .searchResults, .seeAllStops, .currentLocation:
+        case .recentSearches, .searchResults, .seeAllStops, .currentLocation, .favorites:
             itemType = sections[indexPath.section].items[indexPath.row]
         }
         
         switch itemType {
         case .seeAllStops:
             didSelectAllStops = true
-            allStopsTVC.allStops = getAllBusStops()
+            allStopsTVC.allStops = FetchBusStops.shared.getAllStops()
             allStopsTVC.unwindAllStopsTVCDelegate = self
         case .busStop(let busStop):
-            if busStop.name != "Current Location" {
-                insertRecentLocation(location: busStop)
+            if busStop.name != "Current Location" && busStop.name != "Add Your First Favorite!" {
```

Comment:
Maybe abstract this string out in a variable, in case you want to change the phrasing